### PR TITLE
Fix/adapt create assets mutations to format of yaml file

### PR DESCRIPTION
**Pull request by elyousfi5**

### CODE REVIEW

File: src/ostorlab/serve_app/types.py [hunk 1]
```diff
@@ -735,10 +718,10 @@ class OxoLinkInputType(graphene.InputObjectType):
...
 
 class OxoAssetInputType(graphene.InputObjectType):
-    android_file = OxoAndroidFileAssetInputType()
-    ios_file = OxoIOSFileAssetInputType()
-    android_store = OxoAndroidStoreAssetInputType()
-    ios_store = OxoIOSStoreAssetInputType()
+    android_file = graphene.List(OxoAndroidFileAssetInputType)
+    ios_file = graphene.List(OxoIOSFileAssetInputType)
+    android_store = graphene.List(OxoAndroidStoreAssetInputType)
+    ios_store = graphene.List(OxoIOSStoreAssetInputType)
```
Comment (mohsinenar) [inline src/ostorlab/serve_app/types.py:724]: the name should indicate it's a list: android_files, ios_file...

Comment (elyousfi5) [reply src/ostorlab/serve_app/types.py:724]: im adapting to yaml file format : 
image

File: src/ostorlab/serve_app/types.py [hunk 2]
```diff
@@ -735,10 +718,10 @@ class OxoLinkInputType(graphene.InputObjectType):
 
 
 class OxoAssetInputType(graphene.InputObjectType):
-    android_file = OxoAndroidFileAssetInputType()
-    ios_file = OxoIOSFileAssetInputType()
-    android_store = OxoAndroidStoreAssetInputType()
-    ios_store = OxoIOSStoreAssetInputType()
+    android_apk_file = graphene.List(OxoAndroidFileAssetInputType)
```
Comment (3asm) [inline src/ostorlab/serve_app/types.py:721]: The we are missing android_aab_file

File: src/ostorlab/serve_app/types.py [hunk 3]
```diff
@@ -735,10 +718,10 @@ class OxoLinkInputType(graphene.InputObjectType):
...
+    ios_file = graphene.List(OxoIOSFileAssetInputType)
```
Comment (3asm) [inline src/ostorlab/serve_app/types.py:722]: Isn't this ios_ipa_file?

File: src/ostorlab/serve_app/types.py [hunk 4]
```diff
@@ -735,10 +718,10 @@ class OxoLinkInputType(graphene.InputObjectType):
...
+    android_store = graphene.List(OxoAndroidStoreAssetInputType)
+    ios_store = graphene.List(OxoIOSStoreAssetInputType)
```
Comment (3asm) [inline src/ostorlab/serve_app/types.py:724]: Link and IP? are the names corrects?

**Review by 3asm — CHANGES_REQUESTED** [review]

File: src/ostorlab/serve_app/types.py (hunk 4 above)
Comment (elyousfi5) [reply src/ostorlab/serve_app/types.py:724]: here are the names: 
image

File: src/ostorlab/serve_app/types.py (hunk 3 above)
Comment (elyousfi5) [reply src/ostorlab/serve_app/types.py:722]: Nope, its ios_file: 
image

File: src/ostorlab/serve_app/types.py (hunk 2 above)
Comment (elyousfi5) [reply src/ostorlab/serve_app/types.py:721]: the name is just to align with the yaml file, in the db its named android_file and we store there all android files (apk, aab, xapk)

the logic to separate between them is in runScan : 

image

Comment (elyousfi5) [reply src/ostorlab/serve_app/types.py:721]: added

**Review by 3asm — APPROVED** [review]

**Pull request merged**